<?xml version="1.0" encoding="UTF-8" standalone="no"?><testsuite errors="3" failures="0" name="com.studentapp.junit.studentsinfo.FirstSerenityTest" skipped="2" tests="6" time="1.27" timestamp="2023-04-21 05:52:25"><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This is a skipped test"><skipped/></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This is atest with error"><error message="/ by zero" type="java.lang.ArithmeticException">/ by zero</error><system-err>/ by zero&#13;
com.studentapp.junit.studentsinfo.FirstSerenityTest.thisIsAtestWithError(FirstSerenityTest.java:72)&#13;
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)&#13;
</system-err></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This is a pending test"><skipped/></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="This isa failing"><error message="class &quot;org.hamcrest.Matchers&quot;'s signer information does not match signer information of other classes in the same package" type="java.lang.SecurityException">class "org.hamcrest.Matchers"'s signer information does not match signer information of other classes in the same package</error><system-err>class "org.hamcrest.Matchers"'s signer information does not match signer information of other classes in the same package&#13;
io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)&#13;
io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)&#13;
com.studentapp.junit.studentsinfo.FirstSerenityTest.thisIsaFailing(FirstSerenityTest.java:54)&#13;
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)&#13;
</system-err></testcase><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="Get all students"/><testcase classname="com.studentapp.junit.studentsinfo.FirstSerenityTest" name="File dose not exist"><error message="E:\file.txt (The system cannot find the file specified)" type="java.io.FileNotFoundException">E:\file.txt (The system cannot find the file specified)</error><system-err>E:\file.txt (The system cannot find the file specified)&#13;
com.studentapp.junit.studentsinfo.FirstSerenityTest.fileDoseNotExist(FirstSerenityTest.java:78)&#13;
org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)&#13;
org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)&#13;
</system-err></testcase></testsuite>